manager-bundle/src/Command/DebugPluginsCommand.php

```diff
@@ -131,16 +131,16 @@ private function listBundles(): int
             $class = \get_class($bundle);
 
             if (ContaoModuleBundle::class === $class) {
-                $path = sprintf('system/modules/%s', $name);
+                $path = Path::join('system/modules', $name);
             } else {
                 $reflection = new \ReflectionClass($class);
 
-                if (is_dir($dir = \dirname($reflection->getFileName()).'/Resources/contao')) {
-                    $path = (new Filesystem())->makePathRelative($dir, $this->kernel->getProjectDir());
+                if (is_dir($dir = Path::join($reflection->getFileName(), '../Resources/contao'))) {
+                    $path = Path::makeRelative($dir, $this->kernel->getProjectDir());
                 }
             }
 
-            $rows[] = [$bundle->getName(), $path];
+            $rows[] = [$bundle->getName(), $path.'/'];
```

Comment:
Why the trailing slash?

Reply:
Because currently paths in the console output are listed with a trailing slash. We could also remove it + adjust the unit tests.

Reply:
I’d vote for removing it.

Reply:
Me too.

Reply:
Changed in c235e49481be2565fd05dbcb60c9df0a853c5012.